packages/host-inventory/types/index.ts

```diff
@@ -2358,6 +2259,174 @@ export interface SystemProfileThirdPartyServicesCrowdstrike {
+     * @memberof SystemProfileWorkloads
+     */
+    'crowdstrike'?: SystemProfileThirdPartyServicesCrowdstrike;
+    /**
+     *
+     * @type {SystemProfileWorkloadsIbmDb2}
+     * @memberof SystemProfileWorkloads
+     */
+    'ibm_db2'?: SystemProfileWorkloadsIbmDb2;
+    /**
+     *
+     * @type {SystemProfileWorkloadsIntersystems}
+     * @memberof SystemProfileWorkloads
+     */
+    'intersystems'?: SystemProfileWorkloadsIntersystems;
+    /**
+     *
+     * @type {SystemProfileMssql}
+     * @memberof SystemProfileWorkloads
+     */
+    'mssql'?: SystemProfileMssql;
+    /**
+     *
+     * @type {SystemProfileWorkloadsOracleDb}
+     * @memberof SystemProfileWorkloads
+     */
+    'oracle_db'?: SystemProfileWorkloadsOracleDb;
+    /**
+     *
+     * @type {SystemProfileWorkloadsRhelAi}
+     * @memberof SystemProfileWorkloads
+     */
+    'rhel_ai'?: SystemProfileWorkloadsRhelAi;
+    /**
+     *
+     * @type {SystemProfileSap}
+     * @memberof SystemProfileWorkloads
+     */
+    'sap'?: SystemProfileSap;
+}
+/**
+ * Object containing data specific to the IBM DB2 workload
+ * @export
+ * @interface SystemProfileWorkloadsIbmDb2
+ */
+export interface SystemProfileWorkloadsIbmDb2 {
+    /**
+     * Indicates if IBM DB2 is running on the system
+     * @type {boolean}
+     * @memberof SystemProfileWorkloadsIbmDb2
+     */
+    'is_running'?: boolean;
+}
+/**
+ * Object containing data specific to InterSystems workload
+ * @export
+ * @interface SystemProfileWorkloadsIntersystems
+ */
+export interface SystemProfileWorkloadsIntersystems {
+    /**
+     * Indicates if InterSystems is installed on the system
+     * @type {boolean}
+     * @memberof SystemProfileWorkloadsIntersystems
+     */
+    'is_intersystems'?: boolean;
+    /**
+     *
+     * @type {Array<Items>}
+     * @memberof SystemProfileWorkloadsIntersystems
+     */
+    'running_instances'?: Array<Items>;
+}
+/**
+ * Object containing data specific to the Oracle DB workload
+ * @export
+ * @interface SystemProfileWorkloadsOracleDb
+ */
+export interface SystemProfileWorkloadsOracleDb {
+    /**
+     * Indicates if Oracle DB is running on the system
+     * @type {boolean}
+     * @memberof SystemProfileWorkloadsOracleDb
+     */
+    'is_running'?: boolean;
+}
+/**
+ * Object containing information about RHEL AI
+ * @export
+ * @interface SystemProfileWorkloadsRhelAi
+ */
+export interface SystemProfileWorkloadsRhelAi {
+    /**
+     * RHEL AI VARIANT
+     * @type {string}
+     * @memberof SystemProfileWorkloadsRhelAi
+     */
+    'variant'?: string;
+    /**
+     * RHEL AI VERSION ID
+     * @type {string}
+     * @memberof SystemProfileWorkloadsRhelAi
+     */
+    'rhel_ai_version_id'?: string;
+    /**
+     * The list of GPU models on the host
+     * @type {Array<SystemProfileWorkloadsRhelAiGpuModelsInner>}
+     * @memberof SystemProfileWorkloadsRhelAi
+     */
+    'gpu_models'?: Array<SystemProfileWorkloadsRhelAiGpuModelsInner>;
+    /**
```

Comment:
The array translation is happening as expected for other fields, for example. It's just not able to translate the `running_instances` properly, as it seems.